src/main/kotlin/step1/StringAddCalculator.kt

```diff
@@ -0,0 +1,78 @@
+package step1
+
+class StringAddCalculator(
+    vararg delimiter: String = DEFAULT_DELIMITERS
+) {
+    companion object {
+        private val DEFAULT_DELIMITERS = arrayOf(",", ":")
+        private const val REGEX_DELIMITER = "|"
+        private const val REGEX_PREFIX = "("
+        private const val REGEX_POSTFIX = ")"
+        private const val REGEX_INPUT_PARSING = "//(.*)\n(.*)"
+    }
+
+    private val delimiterRegex: Regex
+
+    init {
+        delimiterRegex = delimiterRegex(listOf(*delimiter))
```

Comment:
이런 경우라면 init block에 정의하지 않고 아래처럼 바로 할당하면 어떨까요?

```suggestion
    private val delimiterRegex: Regex = delimiterRegex(listOf(*delimiter))
```

Reply:
맞네요. 감사합니다 ㅠㅠ

Suggested change 어떻게 남기셨어요?

개인적으로 궁금합니다. ㅠㅠ